CHANGELOG.md

```diff
@@ -49,6 +49,27 @@
   });
   ```
   - Replace deprecated `BeforeScreenshotCallback` with new `BeforeCaptureCallback`.
+- Support custom `Sentry.runZoneGuarded` zone creation ([#2088](https://github.com/getsentry/sentry-dart/pull/2088))
+  - Sentry will not create a custom zone anymore if it is started within a custom one.
```

Comment:
maybe also makes sense to mention that this fixes Zone missmatch errors when trying to initialize WidgetsBinding before Sentry on Flutter Web